contrib/adsk/libraries/adsklib/adsklib_3dwood_defs.mtlx

```diff
@@ -0,0 +1,261 @@
+    <input name="color" type="color3" value="0.0, 0.0, 0.0" xpos="-24.456522" ypos="-59.017242"/>
+    <output name="ray_color" type="color3"  />
+    <output name="ray_weight" type="float" />
+  </nodedef>
+
+  <!--
+    Node: <wood3d_util_pore_color>
+    Compute pore color.
+  -->
+  <nodedef name="ND_wood3d_util_pore_color" node="wood3d_util_pore_color">
+    <input name="pore_color_power" type="float" value="0.0" />
+    <input name="pore_weight" type="float" value="0.0"/>
+    <input name="color" type="color3" value="0, 0, 0"/>
+    <output name="pore_color" type="color3"/>
+  </nodedef>
+
+  <!--
+    Node: <wood3d_util_roughness>
+    Compute wood roughness
+  -->
+  <nodedef name="ND_wood3d_util_roughness" node="wood3d_util_roughness" >
+    <input name="use_groove_roughness" type="boolean" value="true" xpos="-72.840576" ypos="-55.172413" />
+    <input name="groove_roughness" type="float" value="0.0" xpos="-79.594200" ypos="-51.879311" />
+    <input name="earlywood_ratio" type="float" value="0.0" xpos="-79.521736" ypos="-50.913792" />
+    <input name="ray_weight" type="float" value="0.0" xpos="-70.753624" ypos="-50.913792" />
+    <input name="roughness" type="float" value="0.0" xpos="-75.702896" ypos="-53.810345" />
+    <input name="ray_roughness" type="float" value="0.0" xpos="-70.869568" ypos="-49.913792" />
+    <output name="wood_roughness" type="float"/>
+  </nodedef>
+
+  <!--
+    Node: <wood3d_util_bump>
+    Compute wood bump. Not working due to the issue of procedural normal map in MaterialX
+  -->
+  <nodedef name="ND_wood3d_util_bump" node="wood3d_util_bump">
+    <input name="use_pores_bump" xpos="-78.398552" ypos="-34.534481" type="boolean" value="false"/>
+    <input name="use_late_wood_bump" xpos="-74.442032" ypos="-30.836206" type="boolean" value="false"/>
+    <input name="late_wood_bump_depth" xpos="-75.405800" ypos="-26.758621" type="float"  value="0.0" />
+    <input name="pore_depth" xpos="-87.246376" ypos="-37.732758"  type="float" value="0.0"/>
+    <input name="earlywood_ratio" type="float" value="0.0" xpos="-80.492752" ypos="-29.241379" />
+    <input name="pore_bump_weight" type="float" value="0.0"  xpos="-88.405800" ypos="-33.931034"/>
+    <output name="normal" type="vector3" />
+  </nodedef>
+
+  <!-- 
+      ==============================================
+      Nodedefs for Autodesk 3D wood graph 
+      ==============================================
+  -->
+  <!--
+    Node: <wood3d>
+    3d wood material node graph
+  -->
+  <nodedef name="ND_wood3d" node="wood3d" nodegroup="texture3d">
+    <input name="seed" uiname="Wood generation seed" uifolder="Wood Space Settings" type="float" value="2" uimin="0" uisoftmax="4096" xpos="-107.971016" ypos="-45.672413" />
+    <input name="scale" unittype="distance" uivisible="false" uiname="Scale (should not be changed by user)" uifolder="Wood Space Settings" type="float" value="0.1" uimin="0" uisoftmax="10" xpos="-110.057968" ypos="-47.879311" />
```

Comment:
I would remove "(should not be changed by the user)". It's legit for the user to adjust the overall size if desired.
Why is the default 0.1? It should be 1.0.
I verified that with scale 1.0 and units in cm, the rings are correctly spaced.